### PR TITLE
Add more colors to fstab syntax

### PR DESCRIPTION
**Pull request by eth-p**

### CODE REVIEW

**Comment by sharkdp** [comment]
Works (and looks) great, thank you.

File: assets/syntaxes/Fstab.sublime-syntax [hunk 1]
```diff
@@ -8,18 +8,126 @@ scope: source.fstab
...
+      set: fstab_pass
+    - match: '\s*[^\s]+\s*'
+      comment: invalid dump field
+      scope: invalid.illegal
+      set: fstab_pass
+    - include: clear
+
+  fstab_pass:
+    - include: comment
+    - match: '\s*[01]\s*'
```
Comment (sharkdp) [inline assets/syntaxes/Fstab.sublime-syntax]: only noticed this after merging, but a `2` also seems to be a valid value here.

Comment (eth-p) [reply assets/syntaxes/Fstab.sublime-syntax]: Whoops. Copy paste got me there. Do you want me to make a PR to fix that, or should one of us just fix and commit directly to master?

Comment (sharkdp) [reply assets/syntaxes/Fstab.sublime-syntax]: No worries :smile:. I think I disabled direct pushes to `master`, so we need a PR.

**Pull request merged**